android/expoview/src/main/java/host/exp/exponent/AppLoader.java

```diff
@@ -123,17 +123,8 @@ public void onCompleted(JSONObject manifest) {
           onError(e);
         }
 
-        if (Constants.isStandaloneApp()) {
-          // in shell/detached apps with SDK <26, we should default to 0 timeout to not introduce a breaking change
-          if (manifestSdkVersion != null) {
-            if (ABIVersion.toNumber(manifestSdkVersion) < ABIVersion.toNumber("26.0.0")) {
-              fallbackToCacheTimeout = DEFAULT_TIMEOUT_LENGTH_BEFORE_SDK26;
-            }
-          }
-        } else {
-          // only support checkAutomatically: never in shell & detached apps
-          shouldCheckForUpdate = true;
-        }
+        // only support checkAutomatically: never in shell & detached apps
+        shouldCheckForUpdate = true;
```

Comment:
this still needs to be wrapped in `if (!Constants.isStandaloneApp())` I believe, unless I'm missing something

Reply:
Phew, thanks for catching that!